Makefile

```diff
@@ -1,14 +1,45 @@
-.PHONY: build run defult
+.PHONY: build run default debug test doc upstream downstream
 
 IMAGE_NAME = modularitycontainers/memcached
```

Comment:
this should be dynamic: but depends whether you want to use this makefile in downstream